src/poetry/factory.py

```diff
@@ -65,7 +63,12 @@ def create_poetry(
         )
 
         # Loading global configuration
-        config = self.create_config(io)
+        with warnings.catch_warnings():
+            # this is preserved to ensure export plugin tests pass in ci,
+            # once poetry-plugin-export version is updated to use one that do not
+            # use Factory.create_config(), this can be safely removed.
+            warnings.filterwarnings("ignore", category=DeprecationWarning)
+            config = self.create_config()
```

Comment:
Since the export plugin test suite mocks Factory.create_config() to inject config into test poetry instance, we need to use this. Otherwise it's back to sed patch.